src/CoreVideo/CVPixelFormatType.cs

```diff
@@ -121,4 +127,33 @@ public enum CVPixelFormatType : uint {
 		// iOS 14.2
 		CV64RgbaLE = 0x6C363472,
 	}
+
+#if !COREBUILD
+	public static class CVPixelFormatTypeExtensions {
+
+#if NET
+		[SupportedOSPlatform ("ios15.0")]
+		[SupportedOSPlatform ("tvos15.0")]
+		[SupportedOSPlatform ("macos12.0")]
+		[UnsupportedOSPlatform ("maccatalyst")]
+		[UnsupportedOSPlatform ("watchos")]
+#else
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), NoMacCatalyst]
+#endif
+		[DllImport (Constants.CoreVideoLibrary)]
```

Comment:
double check the native return type
you might need a `[return: MarshalAs (UnmanagedType.I1)]` to correctly convert the native `bool`